### DISCUSSION TITLE
First run/debug/build gives errors

### DISCUSSION THREAD

**Discussion by Jay-o-Way** [character-map-uwp/Character-Map-UWP#271] [Q&A]
Just forked & cloned it. Opened in Visual Studio. First run gives me errors.
![image](https://github.com/character-map-uwp/Character-Map-UWP/assets/65828559/772cc544-4502-494d-b34c-897e46d5d636)

**Comment by JohnnyWestlake** [comment]
Looks like you can't build the C++/CX project. You'll need to to make sure you have the relevant SDK's installed with C++/CX support (and run the NuGet package restore on the project if you have the SDK's)
